README.md

```diff
@@ -28,97 +28,18 @@ limitations under the License.
 Elyra is a set of AI-centric extensions to JupyterLab Notebooks.
 
 Elyra currently includes:
-* [Notebook Pipelines visual editor](#notebook-pipelines-visual-editor)
-* [Ability to run a notebook as a batch job](#ability-to-run-a-notebook-as-a-batch-job)
-* [Hybrid runtime support](#hybrid-runtime-support)
-* [Python script execution capabilities within the editor](#python-script-execution-support)
-* [Reusable Code Snippets (Experimental)](#reusable-code-snippets-experimental)
-* [Notebook versioning based on git integration](#notebook-versioning-based-on-git-integration)
-* [Notebook navigation using auto-generated **Table of Contents**](#notebook-navigation-using-auto-generated-table-of-contents)
-* [Reusable configuration for runtimes](#reusable-configuration-for-runtimes)
-
+* Notebook Pipelines visual editor
```

Comment:
Just noting here: We'll want to add a link to the readthedocs page once this lands and readthedocs is configured.  If there's a badge for that, that would probably be good as well.

Since this is based on jupyterlab's docs, we should add a similar readthedocs.yml and requirements.txt file to this repo.  The former goes in the repo's root, the latter in the docs root.

We should also make building the docs a step in the [github workflow](https://github.com/elyra-ai/elyra/blob/master/.github/workflows/pythonapp.yml) - probably after build but before test.

Reply:
There is a badge for it. We will need someone with elyra github organization access to setup readthedocs though and then can add the badge after.  I think there may be some additional work here to make that work. I quickly tried using my forked repo to see the process and it didn't work so nicely, but maybe that is because of the missing requirements.txt and readthedocs.yml file. I'll take a look at cleaning that up and see if anything else is needed.

I'll also take a look at the workflows. Never looked at those before.

Reply:
Addressed the readthedocs.yml and requirements.txt files. I also commited some changes to get it to work with readthedocs builds.  A preview is here: https://kjdoyle-elyra.readthedocs.io/en/latest/ 

Will continue to look at workflows next.

Reply:
Added github workflow for docs.